integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiCustomSslStore.java

```diff
@@ -111,18 +110,18 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
     // install and verify operator
     installAndVerifyOperator(opNamespace, domainNamespace);
 
-    // create secret for admin credentials
+    // create secret for admin credential with special characters
     logger.info("Create secret for admin credentials");
     String adminSecretName = "weblogic-credentials";
     assertDoesNotThrow(() -> createDomainSecret(adminSecretName, 
-            ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT, domainNamespace),
+            ADMIN_USERNAME_DEFAULT, "W##%*}!`''1~3x", domainNamespace),
```

Comment:
can you add all the characters mentioned in the JIRA - Testing a single quote, a double quote, two single quotes, two double quotes, single slash, back slash, double slash, ‘$’, back-quote, double back-quote, ‘!’, etc, etc...

Reply:
done, Now new password is [ W##%*}!""`'']\\//1$$~x ]

Reply:
This is one-off variable, not being used in multiple classes.